api/storage_service.cc

```diff
@@ -42,6 +42,8 @@
 #include "database.hh"
 #include "db/extensions.hh"
 
+extern logging::logger apilog;
```

Comment:
It looks like we don't use this anywhere.

Reply:
right, done